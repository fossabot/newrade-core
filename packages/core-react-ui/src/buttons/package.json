{
  "name": "@newrade/core-react-ui-buttons",
  "version": "1.0.0",
  "description": "Flexible button components",
  "author": {
    "email": "info@newrade.ca",
    "name": "Newrade",
    "url": "https://github.com/newrade/newrade"
  },
  "homepage": "https://github.com/newrade/newrade-core",
  "repository": {
    "directory": "packages/core-react-ui/src/buttons",
    "type": "git",
    "url": "git+https://github.com/newrade/newrade-core.git"
  },
  "keywords": [
    "button",
    "button-group",
    "react",
    "swadloon-ui",
    "component"
  ],
  "license": "MIT",
  "bugs": {
    "url": "https://github.com/newrade/newrade-core/issues"
  },
  "main": "dist/index.js",
  "module": "dist/index.js",
  "files": [
    "dist",
    "src"
  ],
  "scripts": {
    "build": "yarn build:lib",
    "build:lib": "yarn tsc:types && yarn swc:build",
    "clean": "yarn clean:lib",
    "clean:dist": "rimraf -rf dist/**",
    "clean:lib": "yarn clean:dist && yarn clean:tsc",
    "clean:tsc": "rimraf -f ./**/*.tsbuildinfo *.tsbuildinfo",
    "prestart": "yarn build",
    "start": "yarn start:lib",
    "start:lib": "concurrently \"yarn tsc:types:watch\" \"yarn swc:build:watch\"",
    "swc:build": "swc ./src --config-file ../../../../.swcrc --source-maps --ignore **/__test__/**/* --out-dir dist",
    "swc:build:watch": "yarn swc:build --watch",
    "test": "yarn test:lib",
    "test:lib": "cross-env NODE_OPTIONS=--experimental-vm-modules jest --passWithNoTests",
    "test:lib:watch": "yarn test:lib --watch",
    "test:watch": "yarn test:lib:watch",
    "tsc:typecheck": "tsc --noEmit --emitDeclarationOnly false",
    "tsc:types": "yarn tsc -b",
    "tsc:types:watch": "yarn tsc:types --watch --preserveWatchOutput"
  },
  "sideEffects": false,
  "types": "dist/index.cjs.d.ts",
  "dependencies": {
    "@newrade/core-design-system": "*",
    "@newrade/core-react-ui-utilities-test": "*",
    "@newrade/core-types": "*",
    "@vanilla-extract/css": "^1.6.8",
    "@vanilla-extract/recipes": "^0.2.3",
    "react": "18.0.0",
    "react-dom": "18.0.0"
  }
}
