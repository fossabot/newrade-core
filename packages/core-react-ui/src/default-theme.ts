/**
 *
 * Export the default theme values
 *
 */

export * from './default-theme/default-animations';
export * from './default-theme/default-buttons';
export * from './default-theme/default-code-colors';
export * from './default-theme/default-code-colors-dark';
export * from './default-theme/default-colors';
export * from './default-theme/default-colors-dark';
export * from './default-theme/default-effects';
export * from './default-theme/default-fonts';
export * from './default-theme/default-iconography';
export * from './default-theme/default-layout';
export * from './default-theme/default-sizing';
export * from './default-theme/default-typography';
