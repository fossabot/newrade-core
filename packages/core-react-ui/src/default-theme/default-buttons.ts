import { BorderStyles, Buttons } from '@newrade/core-design-system';

import { defaultColorIntents, defaultColors } from './default-colors';

export const defaultButtonBorder: BorderStyles = {
  default: {
    width: 1,
    style: 'solid',
    color: defaultColors.colorIntents.primary,
    radius: {
      all: 3,
    },
  },
  top: {},
  right: {},
  bottom: {},
  left: {},
};

export const defaultButtons: Buttons = {
  sizes: {
    large: {
      padding: {
        default: NaN,
        left: 16,
        right: 16,
        top: NaN,
        bottom: NaN,
      },
      iconPadding: 12,
      iconOnlyPadding: 6,
      border: {
        ...defaultButtonBorder,
        default: {
          ...defaultButtonBorder.default,
          radius: {
            all: 4,
          },
        },
      },
      outline: {
        color: defaultColorIntents.primary,
        width: defaultButtonBorderStyle.width,
        radius: defaultButtonBorderStyle.radius,
        style: defaultButtonBorderStyle.style,
      },
      iconSize: 24,
      iconMargin: {
        left: 10,
        right: 10,
        top: NaN,
        bottom: NaN,
      },
      border: defaultButtonBorder,
      outline: {},
      iconSize: 16,
      iconMargin: {},
    },
    medium: {
      padding: {
        default: NaN,
        left: 10,
        right: 10,
        top: NaN,
        bottom: NaN,
      },
      iconPadding: 10,
      iconOnlyPadding: 6,
      border: {
        ...defaultButtonBorder,
        default: {
          ...defaultButtonBorder.default,
          radius: {
            all: 4,
          },
        },
      },
      outline: {
        color: defaultColorIntents.primary,
        width: defaultButtonBorderStyle.width,
        radius: defaultButtonBorderStyle.radius,
        style: defaultButtonBorderStyle.style,
      },
      border: defaultButtonBorder,
      outline: {},
      iconSize: 16,
      iconMargin: {},
    },
    small: {
      padding: {
        default: NaN,
        left: 8,
        right: 8,
        top: NaN,
        bottom: NaN,
      },
      iconPadding: 8,
      iconOnlyPadding: 6,
      border: {
        ...defaultButtonBorder,
        default: {
          ...defaultButtonBorder.default,
          radius: {
            all: 3,
          },
        },
      },
      outline: {
        color: defaultColorIntents.primary,
        width: defaultButtonBorderStyle.width,
        radius: defaultButtonBorderStyle.radius,
        style: defaultButtonBorderStyle.style,
      },
      border: defaultButtonBorder,
      outline: {},
      iconSize: 16,
      iconMargin: {},
    },
    xSmall: {
      padding: {
        default: NaN,
        left: 6,
        right: 6,
        top: NaN,
        bottom: NaN,
      },
      iconPadding: 6,
      iconOnlyPadding: 6,
      border: {
        ...defaultButtonBorder,
        default: {
          ...defaultButtonBorder.default,
          radius: {
            all: 3,
          },
        },
      },
      outline: {
        color: defaultColorIntents.primary,
        width: defaultButtonBorderStyle.width,
        radius: defaultButtonBorderStyle.radius,
        style: defaultButtonBorderStyle.style,
      },
      border: defaultButtonBorder,
      outline: {},
      iconSize: 16,
      iconMargin: {},
    },
  },
  variants: {
    primary: {
      textColor: defaultColorIntents.primaryText,
      backgroundColor: defaultColorIntents.primaryText,
      iconColor: defaultColorIntents.primaryText,
      borderColor: defaultColorIntents.primaryText,
      outlineColor: defaultColorIntents.primaryText,
    },
    secondary: {
      textColor: defaultColorIntents.primaryText,
      backgroundColor: defaultColorIntents.primaryText,
      iconColor: defaultColorIntents.primaryText,
      borderColor: defaultColorIntents.primaryText,
      outlineColor: defaultColorIntents.primaryText,
    },
    tertiary: {
      textColor: defaultColorIntents.primaryText,
      backgroundColor: defaultColorIntents.primaryText,
      iconColor: defaultColorIntents.primaryText,
      borderColor: defaultColorIntents.primaryText,
      outlineColor: defaultColorIntents.primaryText,
    },
    positive: {
      textColor: defaultColorIntents.primaryText,
      backgroundColor: defaultColorIntents.primaryText,
      iconColor: defaultColorIntents.primaryText,
      borderColor: defaultColorIntents.primaryText,
      outlineColor: defaultColorIntents.primaryText,
    },
    warning: {
      textColor: defaultColorIntents.warningText,
      backgroundColor: defaultColorIntents.primaryText,
      iconColor: defaultColorIntents.primaryText,
      borderColor: defaultColorIntents.primaryText,
      outlineColor: defaultColorIntents.primaryText,
    },
    danger: {
      textColor: defaultColorIntents.dangerText,
      backgroundColor: defaultColorIntents.primaryText,
      iconColor: defaultColorIntents.primaryText,
      borderColor: defaultColorIntents.primaryText,
      outlineColor: defaultColorIntents.primaryText,
    },
  },
};

/**
 * DefaultButtons object without the `vars` property which is used only
 * in application-defined theme configuration
 */
export type DefaultButtons = Omit<Buttons, 'vars'>;
