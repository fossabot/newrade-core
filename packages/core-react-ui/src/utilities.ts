export * from './utilities/colors.utilities';
export * from './utilities/component.utilities';
export * from './utilities/css-variable.utilities';
export * from './utilities/effects.utilities';
export * from './utilities/filter-color.utility';
export * from './utilities/log.utilities';
export * from './utilities/text-capsize.utilities';
export * from './utilities/text.utilities';
export * from './utilities/theme.utilities';
export * from './utilities/utilities';

