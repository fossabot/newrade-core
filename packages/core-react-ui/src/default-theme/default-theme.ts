import { COLOR_SCHEME, DesignSystemV2 } from '@newrade/core-design-system';

import { defaultAnimations } from './default-animations';
import { defaultButtons } from './default-buttons';
import { defaultColors } from './default-colors';
import { defaultEffects } from './default-effects';
import { defaultIconography } from './default-iconography';
import { defaultLayout } from './default-layout';
import { defaultSizing } from './default-sizing';
import { defaultTypography } from './default-typography';

export const defaultTheme: DesignSystemV2 = {
  name: 'core-react-ui',
  colorScheme: COLOR_SCHEME.LIGHT,
  colors: defaultColors,
  effects: defaultEffects,
  sizing: defaultSizing,
  iconography: defaultIconography,
  typography: defaultTypography,
  layout: defaultLayout,
  animations: defaultAnimations,
  components: {
    buttons: defaultButtons,
  },
};
