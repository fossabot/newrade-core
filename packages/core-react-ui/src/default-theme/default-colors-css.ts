import { getCSSColorsV2 } from '../utilities-theme/colors.utilities';

import { defaultColors } from './default-colors';

export const defaultCSSColors = getCSSColorsV2(defaultColors);
