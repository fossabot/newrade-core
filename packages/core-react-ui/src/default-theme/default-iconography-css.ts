import { defaultIconography } from '../default-theme';
import { getCSSIconography } from '../utilities-theme/iconography.utilities';

export const defaultCSSIconography = getCSSIconography(defaultIconography);
