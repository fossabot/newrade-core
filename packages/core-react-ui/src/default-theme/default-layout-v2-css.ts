import { defaultLayout } from '../default-theme';
import { getCSSLayoutV2 } from '../utilities-theme/layout.utilities';

export const defaultCSSLayoutV2 = getCSSLayoutV2(defaultLayout);
