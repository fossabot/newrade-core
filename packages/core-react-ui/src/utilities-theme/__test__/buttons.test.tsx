import { defaultTheme } from '../../default-theme/default-theme';
import { getCSSButtons } from '../buttons.utilities';

describe('buttons utilities', () => {
  describe(`${getCSSButtons.name}`, () => {
    it(`should generate CSS variable names for a given Colors object`, () => {
      const buttonsCSS = getCSSButtons(defaultTheme);
      expect(buttonsCSS).toMatchSnapshot();
    });
  });
});
