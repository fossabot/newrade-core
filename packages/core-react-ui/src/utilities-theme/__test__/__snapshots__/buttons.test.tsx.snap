// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`buttons utilities getCSSButtons should generate CSS variable names for a given Colors object 1`] = `
Object {
  "sizes": Object {
    "large": Object {
      "border": Object {
        "bottom": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "default": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "4px",
          "style": "solid",
          "width": "1px",
        },
        "left": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "right": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "top": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
      },
      "height": "56px",
      "iconMargin": Object {
        "bottom": "",
        "left": "10px",
        "right": "10px",
        "top": "",
      },
      "iconSize": "24px",
      "outline": Object {
        "color": "hsl(222, 50%, 50%, 1)",
        "radius": "3px",
        "style": "solid",
        "width": "1px",
      },
      "padding": Object {
        "bottom": "",
        "default": "",
        "left": "16px",
        "right": "16px",
        "top": "",
      },
      "width": "",
    },
    "medium": Object {
      "border": Object {
        "bottom": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "default": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "4px",
          "style": "solid",
          "width": "1px",
        },
        "left": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "right": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "top": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
      },
      "height": "40px",
      "iconMargin": Object {
        "bottom": "",
        "left": "10px",
        "right": "10px",
        "top": "",
      },
      "iconSize": "16px",
      "outline": Object {
        "color": "hsl(222, 50%, 50%, 1)",
        "radius": "3px",
        "style": "solid",
        "width": "1px",
      },
      "padding": Object {
        "bottom": "",
        "default": "",
        "left": "10px",
        "right": "10px",
        "top": "",
      },
      "width": "",
    },
    "small": Object {
      "border": Object {
        "bottom": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "default": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "left": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "right": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "top": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
      },
      "height": "32px",
      "iconMargin": Object {
        "bottom": "",
        "left": "10px",
        "right": "10px",
        "top": "",
      },
      "iconSize": "16px",
      "outline": Object {
        "color": "hsl(222, 50%, 50%, 1)",
        "radius": "3px",
        "style": "solid",
        "width": "1px",
      },
      "padding": Object {
        "bottom": "",
        "default": "",
        "left": "8px",
        "right": "8px",
        "top": "",
      },
      "width": "",
    },
    "xSmall": Object {
      "border": Object {
        "bottom": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "default": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "left": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "right": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
        "top": Object {
          "color": "hsl(222, 50%, 50%, 1)",
          "radius": "3px",
          "style": "solid",
          "width": "1px",
        },
      },
      "height": "24px",
      "iconMargin": Object {
        "bottom": "",
        "left": "10px",
        "right": "10px",
        "top": "",
      },
      "iconSize": "16px",
      "outline": Object {
        "color": "hsl(222, 50%, 50%, 1)",
        "radius": "3px",
        "style": "solid",
        "width": "1px",
      },
      "padding": Object {
        "bottom": "",
        "default": "",
        "left": "6px",
        "right": "6px",
        "top": "",
      },
      "width": "",
    },
  },
  "variants": Object {
    "danger": Object {
      "backgroundColor": "hsl(360, 84%, 48%, 1)",
      "backgroundColorActive": "hsl(360, 84%, 27%, 1)",
      "backgroundColorDisabled": "hsl(222, 6%, 90%, 1)",
      "borderColor": "hsl(360, 84%, 48%, 1)",
      "borderColorActive": "hsl(360, 84%, 48%, 1)",
      "borderColorDisabled": "hsl(222, 6%, 90%, 1)",
      "iconColor": "hsl(222, 10%, 100%, 1)",
      "iconColorActive": "hsl(222, 10%, 100%, 1)",
      "iconColorDisabled": "hsl(222, 10%, 100%, 1)",
      "outlineColor": "hsl(360, 84%, 48%, 1)",
      "outlineColorActive": "hsl(360, 84%, 48%, 1)",
      "outlineColorDisabled": "hsl(360, 84%, 48%, 1)",
      "textColor": "hsl(222, 10%, 100%, 1)",
      "textColorActive": "hsl(222, 10%, 100%, 1)",
      "textColorDisabled": "hsl(222, 6%, 73%, 1)",
    },
    "positive": Object {
      "backgroundColor": "hsl(112, 71%, 53%, 1)",
      "backgroundColorActive": "hsl(112, 71%, 32%, 1)",
      "backgroundColorDisabled": "hsl(222, 6%, 90%, 1)",
      "borderColor": "hsl(112, 71%, 53%, 1)",
      "borderColorActive": "hsl(112, 71%, 53%, 1)",
      "borderColorDisabled": "hsl(222, 6%, 90%, 1)",
      "iconColor": "hsl(222, 10%, 100%, 1)",
      "iconColorActive": "hsl(222, 10%, 100%, 1)",
      "iconColorDisabled": "hsl(222, 10%, 100%, 1)",
      "outlineColor": "hsl(112, 71%, 53%, 1)",
      "outlineColorActive": "hsl(112, 71%, 53%, 1)",
      "outlineColorDisabled": "hsl(112, 71%, 53%, 1)",
      "textColor": "hsl(222, 10%, 100%, 1)",
      "textColorActive": "hsl(222, 10%, 100%, 1)",
      "textColorDisabled": "hsl(222, 6%, 73%, 1)",
    },
    "primary": Object {
      "backgroundColor": "hsl(222, 50%, 50%, 1)",
      "backgroundColorActive": "hsl(222, 50%, 32%, 1)",
      "backgroundColorDisabled": "hsl(222, 6%, 90%, 1)",
      "borderColor": "hsl(222, 50%, 50%, 1)",
      "borderColorActive": "hsl(222, 50%, 50%, 1)",
      "borderColorDisabled": "hsl(222, 6%, 90%, 1)",
      "iconColor": "hsl(222, 10%, 100%, 1)",
      "iconColorActive": "hsl(222, 10%, 100%, 1)",
      "iconColorDisabled": "hsl(222, 10%, 100%, 1)",
      "outlineColor": "hsl(222, 50%, 50%, 1)",
      "outlineColorActive": "hsl(222, 50%, 50%, 1)",
      "outlineColorDisabled": "hsl(222, 50%, 50%, 1)",
      "textColor": "hsl(222, 10%, 100%, 1)",
      "textColorActive": "hsl(222, 10%, 100%, 1)",
      "textColorDisabled": "hsl(222, 6%, 73%, 1)",
    },
    "secondary": Object {
      "backgroundColor": "hsl(0, 0%, 0%, 0)",
      "backgroundColorActive": "hsl(222, 50%, 86%, 1)",
      "backgroundColorDisabled": "hsl(222, 6%, 90%, 1)",
      "borderColor": "hsl(222, 50%, 50%, 1)",
      "borderColorActive": "hsl(222, 50%, 50%, 1)",
      "borderColorDisabled": "hsl(222, 6%, 90%, 1)",
      "iconColor": "hsl(222, 50%, 50%, 1)",
      "iconColorActive": "hsl(222, 50%, 50%, 1)",
      "iconColorDisabled": "hsl(222, 50%, 50%, 1)",
      "outlineColor": "hsl(222, 50%, 50%, 1)",
      "outlineColorActive": "hsl(222, 50%, 50%, 1)",
      "outlineColorDisabled": "hsl(222, 50%, 50%, 1)",
      "textColor": "hsl(222, 50%, 50%, 1)",
      "textColorActive": "hsl(222, 50%, 50%, 1)",
      "textColorDisabled": "hsl(222, 6%, 73%, 1)",
    },
    "tertiary": Object {
      "backgroundColor": "hsl(0, 0%, 0%, 0)",
      "backgroundColorActive": "hsl(222, 50%, 86%, 1)",
      "backgroundColorDisabled": "hsl(222, 6%, 90%, 1)",
      "borderColor": "hsl(0, 0%, 0%, 0)",
      "borderColorActive": "hsl(0, 0%, 0%, 0)",
      "borderColorDisabled": "hsl(222, 6%, 90%, 1)",
      "iconColor": "hsl(222, 50%, 50%, 1)",
      "iconColorActive": "hsl(222, 50%, 50%, 1)",
      "iconColorDisabled": "hsl(222, 50%, 50%, 1)",
      "outlineColor": "hsl(222, 50%, 50%, 1)",
      "outlineColorActive": "hsl(222, 50%, 50%, 1)",
      "outlineColorDisabled": "hsl(222, 50%, 50%, 1)",
      "textColor": "hsl(222, 50%, 50%, 1)",
      "textColorActive": "hsl(222, 50%, 50%, 1)",
      "textColorDisabled": "hsl(222, 6%, 73%, 1)",
    },
    "warning": Object {
      "backgroundColor": "hsl(47, 90%, 54%, 1)",
      "backgroundColorActive": "hsl(47, 90%, 33%, 1)",
      "backgroundColorDisabled": "hsl(222, 6%, 90%, 1)",
      "borderColor": "hsl(47, 90%, 54%, 1)",
      "borderColorActive": "hsl(47, 90%, 54%, 1)",
      "borderColorDisabled": "hsl(222, 6%, 90%, 1)",
      "iconColor": "hsl(222, 10%, 100%, 1)",
      "iconColorActive": "hsl(222, 10%, 100%, 1)",
      "iconColorDisabled": "hsl(222, 10%, 100%, 1)",
      "outlineColor": "hsl(47, 90%, 54%, 1)",
      "outlineColorActive": "hsl(47, 90%, 54%, 1)",
      "outlineColorDisabled": "hsl(47, 90%, 54%, 1)",
      "textColor": "hsl(222, 10%, 100%, 1)",
      "textColorActive": "hsl(222, 10%, 100%, 1)",
      "textColorDisabled": "hsl(222, 6%, 73%, 1)",
    },
  },
}
`;
